src/Monticello/MethodAddition.class.st

```diff
@@ -76,7 +76,7 @@ MethodAddition >> notifyObservers [
 			priorCategoryOrNil = category ifFalse: [
        			SystemAnnouncer uniqueInstance methodRecategorized: compiledMethod oldProtocol: priorCategoryOrNil ] ].
 	"The following code doesn't seem to do anything."
```

Comment:
Maybe remove it if it doesn't do anything?

Reply:
I prefer not to touch that in that PR because I'm not sure it does nothing ahah